packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/CameraInfoFlutterApiImpl.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camerax;
+
+import androidx.camera.core.CameraInfo;
+import io.flutter.plugin.common.BinaryMessenger;
+import io.flutter.plugins.camerax.GeneratedCameraXLibrary.CameraInfoFlutterApi;
+
+public class CameraInfoFlutterApiImpl extends CameraInfoFlutterApi {
+  private final InstanceManager instanceManager;
+
+  public CameraInfoFlutterApiImpl(
+      BinaryMessenger binaryMessenger, InstanceManager instanceManager) {
+    super(binaryMessenger);
+    this.instanceManager = instanceManager;
+  }
+
+  void create(CameraInfo cameraInfo, Reply<Void> reply) {
+    create(instanceManager.getIdentifierForStrongReference(cameraInfo), reply);
```

Comment:
Since this is the create method, shouldn't this add the `cameraInfo` to the `InstanceManager` before you try to send it back.

Reply:
I originally separated the two which is why it's not here: https://github.com/flutter/plugins/pull/6348/files#diff-9a5ea0901ea92ff282ea8c97ec6ab14d43890465693f5916e6f9b59abf5175f8R61. I think moving it here would make more sense though!